src/components/Layouts/Layouts.stories.js

```diff
@@ -0,0 +1,248 @@
+/**
+ * @file Modular layout stories.
+ * @copyright IBM Security 2020
+ */
+
+import { boolean } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/react';
+
+import React from 'react';
+import { Grid, Row, Column } from 'carbon-components-react';
+import ArrowRight20 from '@carbon/icons-react/lib/arrow--right/20';
+
+import {
+  Accordion,
+  AccordionItem,
+  Button,
+  Tab,
+  Tabs,
+  TypeLayout,
+  TypeLayoutBody,
+  TypeLayoutRow,
+  TypeLayoutCell,
+} from '../..';
+
+import { disableCentered, patterns } from '../../../.storybook';
+
+const toggleDebugging = () => boolean('Toggle debugging 🕵️‍♀️', false);
+
+const showAccordion = () => boolean('Show accordion', true);
+const showButtonGroup = () => boolean('Show button group', true);
+const showCardGroup = () => boolean('Show card group', true);
+const showTabGroup = () => boolean('Show tab group', true);
```

Comment:
Some of these don't feel particular valuable to me. With the except of `showTabGroup`, which ends up changing the column span and therefore modifies the layout, toggling on/off these UI groups just removes a column. 🤔